### ISSUE TITLE
TestSys: `add secret map` results in `MethodNotAllowed` error

### ISSUE THREAD

**Issue by mjsterckx** [bottlerocket-os/bottlerocket#2659]
**What I expected to happen:**

Running `cargo make -e TESTSYS_KUBECONFIG="creds.kubeconfig" testsys add secret map   --name "aws-creds"  "access-key-id=$(aws configure get default.aws_access_key_id)"  "secret-access-key=$(aws configure get default.aws_secret_access_key)"` should add a secret to the cluster.

**What actually happened:**

Running the above command results in this error:
```bash
ubuntu@ip-172-31-65-200:~/dev/bottlerocket$ cargo make -e TESTSYS_KUBECONFIG="creds.kubeconfig" testsys add secret map   --name "aws-creds"  "access-key-id=$(aws configure get default.aws_access_key_id)"  "secret-access-key=$(aws configure get default.aws_secret_access_key)"
[cargo-make] INFO - cargo make 0.36.0
[cargo-make] INFO - Build File: Makefile.toml
[cargo-make] INFO - Task: testsys
[cargo-make] INFO - Profile: development
[cargo-make] INFO - Running Task: setup
[cargo-make] INFO - Running Task: fetch-sources
[cargo-make] INFO - Running Task: test-tools
[cargo-make] INFO - Running Task: testsys
[2022-12-13T22:30:03Z ERROR testsys] Error Creating secret: ApiError: the server does not allow this method on the requested resource: MethodNotAllowed (ErrorResponse { status: "Failure", message: "the server does not allow this method on the requested resource", reason: "MethodNotAllowed", code: 405 })
[cargo-make] ERROR - Error while executing command, exit code: 1
[cargo-make] WARN - Build Failed.
```

Running the same command using TestSys from https://github.com/bottlerocket-os/bottlerocket-test-system does not return any errors and the secret gets added successfully.

**How to reproduce the problem:**

Set up a testsys cluster and run `cargo make -e TESTSYS_KUBECONFIG="creds.kubeconfig" testsys add secret map   --name "aws-creds"  "access-key-id=$(aws configure get default.aws_access_key_id)"  "secret-access-key=$(aws configure get default.aws_secret_access_key)"`

**Comment by ecpullen** [comment]
The root of this issue has been fixed in `bottlerocket-test-system` and will be resolved when testsys is updated to v0.0.4

**Comment by ecpullen** [comment]
This was resolved when TestSys was upgraded to v0.0.5
```
$ cargo make testsys add secret map   --name "aws-creds"  "access-key-id=$(aws configure get default.aws_access_key_id)"  "secret-access-key=$(aws configure get default.aws_secret_access_key)"
[cargo-make] INFO - cargo make 0.36.1
[cargo-make] INFO - Build File: Makefile.toml
[cargo-make] INFO - Task: testsys
[cargo-make] INFO - Profile: development
[cargo-make] INFO - Running Task: setup
[cargo-make] INFO - Running Task: fetch-sources
[cargo-make] INFO - Running Task: test-tools
[cargo-make] INFO - Running Task: testsys
Successfully added 'aws-creds' to secrets.
[cargo-make] INFO - Build Done in 27.54 seconds.
```
